auto_round/export/__init__.py

```diff
@@ -12,9 +12,39 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from .register import EXPORT_FORMAT
-from auto_round.export.export_to_autogptq.export import save_quantized_as_autogptq
-from .export_to_itrex import save_quantized_as_itrex, QuantConfig
-from .export_to_autoround.export import save_quantized_as_autoround
-from .export_to_awq.export import save_quantized_as_autoawq
+from auto_round.export.register import EXPORT_FORMAT, register_format
 
+
+@register_format("auto_gptq")
+def _save_quantized_as_autogptq(*args, **kwargs):
```

Comment:
@WeiweiZhang1 Please note that I moved all `register_format` to __init__.py to ensure that only the relevant code is imported.